tests/serial/mapping-nearest-neighbor-gradient/GradientTestBidirectionalReadVector.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<precice-configuration>
+  <solver-interface dimensions="3" experimental="on">
```

Comment:
This was suggested during the review, since we might remove the flag "gradient = on". I wasn't sure if I should apply the attribute here or under the data attribute.